src/utils/chatService.js

```diff
@@ -158,4 +159,20 @@ export default class ChatService {
       callback(null);
     }
   }
+
+  desactivateUpnp = async () => {
```

Comment:
```suggestion
  deactivateUpnp = async () => {
```

Reply:
@xkevin190 if you prefer, use `enable/disable` or `enabled/disabled` for this purpose